crates/iota-data-ingestion-core/src/progress_store/mod.rs

```diff
@@ -82,6 +92,29 @@ impl<P: ProgressStore> ProgressStoreWrapper<P> {
     }
 }
 
+/// A simple, in-memory progress store primarily used for unit testing.
+///
+/// # Note
+///
+/// - It does not provide any save functionality.
+/// - Only load functionality is provided.
```

Comment:
Maybe we should write that it provides save and load, but the save is not persistent?

Reply:
fixed: https://github.com/iotaledger/iota/pull/5735/commits/56106d99cc8f8f9d079a5e9371177a06437cac40#diff-fe85c8baebf1df63db20121a40711b2e88a9bfe28cf2cd71041b22dba5802afaR99